functions/src/admin/AdminPaymentTaskProcessor.ts

```diff
@@ -77,6 +81,64 @@ export class AdminPaymentTaskProcessor {
 		return `Created ${paymentsCreated} payments`;
 	};
 
+	private sendNotifications = async (recipientDocs: QueryDocumentSnapshot<Recipient>[]) => {
+		let newNotificationsSent = 0;
+		let existingNotifications = 0;
+		const now = moment();
+
+		for (const recipientDoc of recipientDocs) {
+			if (recipientDoc.data().test_recipient) continue;
+
+			const paymentDocRef = this.firestoreAdmin.doc<Payment>(
+				`${RECIPIENT_FIRESTORE_PATH}/${recipientDoc.id}/${PAYMENT_FIRESTORE_PATH}`,
+				now.format('YYYY-MM')
+			);
+
+			if ((await paymentDocRef.get()).exists) {
+				const paymentDocSnap = await paymentDocRef.get();
+				const payment: Payment = paymentDocSnap.data() as Payment;
+				if (!payment.message) {
+					const recipient: Recipient = recipientDoc.data();
+					const [messageSid, _messageStatus, messageContent] = await sendSms({
```

Comment:
nit: instead of an Array, I would return an Object in sendSms. Makes it easier to use since one doesn't need to know what the first, second and third element of the array means.

Reply:
very cool idea, added that 👍